app/forms/declaration/additionaldocuments/DocumentsProduced.scala

```diff
@@ -100,21 +102,10 @@ object DocumentsProduced extends DeclarationPage {
 
   def globalErrors(form: Form[DocumentsProduced]): Form[DocumentsProduced] = {
 
-    def validate(docs: DocumentsProduced) =
+    def globalValidate(docs: DocumentsProduced) =
       docs.documentWriteOff.map(wo => DocumentWriteOff.globalErrors(wo)).getOrElse(Seq.empty)
 
-    def withMeasurementUnitGroupErrors(formErrors: Seq[FormError]) =
-      if (formErrors.exists(_.key.endsWith(measurementUnitKey)) && formErrors.exists(_.key.endsWith(qualifierKey))) {
```

Comment:
Got rid of this "feature" where we were replacing two errors for measurement unit and qualifier with a single error as this new error was out of sequence with other errors on the form.

I could have tried to insert the new error in the correct location but displaying both errors is how the "date" field above works if, for example, the month and year fields are missing.